cypher/cypher-prettifier/src/main/scala/org/neo4j/cypher/docgen/tooling/CypherPrettifier.scala

```diff
@@ -38,7 +37,7 @@ case object CypherPrettifier extends Prettifying {
   override def apply(input: String): String = {
     val prettifier: Prettifier = Prettifier(ExpressionStringifier())
 
-    val parsingResults: Statement = JavaCCParser.parse(input, OpenCypherExceptionFactory(None), new AnonymousVariableNameGenerator())
+    val parsingResults: Statement = JavaCCParser.parse(input, OpenCypherExceptionFactory(None))
```

Comment:
This was fixed in PR https://github.com/neo4j/neo4j-documentation/pull/1585/files#diff-5c69dcf2f1ee2f96747ac41d1042c98c8fee5c30f56ca070b6ca382ed2ec5329R41